it/uk/gov/hmrc/agentsubscriptionfrontend/support/BaseISpec.scala

```diff
@@ -85,7 +85,10 @@ abstract class BaseISpec
     status(result) shouldBe OK
     contentType(result) shouldBe Some("text/html")
     charset(result) shouldBe Some("utf-8")
-    expectedSubstrings.foreach(s => bodyOf(result) should include(s))
+    expectedSubstrings.foreach { expectedSubstring =>
```

Comment:
can we store bodyOf(result) in val and check rather than doing it every time in for-each